forge-gui/res/tokenscripts/c_0_2_ornithopter_flying.txt

```diff
@@ -0,0 +1,6 @@
+Name:Ornithopter
```

Comment:
The file for this one just needs to be named ornithopter.txt as it's a "named" token and this is what the edition file shows